pyvrp/GeneticAlgorithm.py

```diff
@@ -157,10 +157,10 @@ def run(self, stop: StoppingCriterion):
             curr_best = self._cost_evaluator.cost(self._best)
 
             parents = self._pop.select(self._rng, self._cost_evaluator)
-            offspring = self._op(
+            offspring = self._crossover(
                 parents, self._data, self._cost_evaluator, self._rng
             )
-            self._search(offspring)
+            self._handle_offspring(offspring)
```

Comment:
Nit: `handle_offspring` is a bit vague. We already assume that the search method improves the solution, so I'd prefer to call this `improve_offspring` (this also better reflects the description of [HGS](https://pyvrp.readthedocs.io/en/latest/setup/introduction_to_hgs.html)). 

Related: we no longer use the concept of 'educating offspring.' There's only one exception: it's still mentioned in the notebook [here](https://github.com/search?q=repo%3APyVRP%2FPyVRP%20educate&type=code). Let's get rid of that there too.

Reply:
Sure thing